src/shared/bot_nav_shared.h

```diff
@@ -86,6 +86,7 @@ struct NavgenConfig {
 	int filterGaps; // boolean - add new walkable spans so bots can walk over small gaps
 	int generatePatchTris; // boolean - generate triangles from the BSP's patches
 	float autojumpSecurity; // percentage - allow to use part of jump magnitude (with default gravity of 800) as stepsize. The result can not excess the agent's height, except if STEPSIZE is already doing it (then STEPSIZE will be used)
+	bool crouchSupport = true;
```

Comment:
`// Part of header which must contain 4-byte members only`

Reply:
@bmorel is this thread solved?

Reply:
Does using `int` instead of `bool` would solve the thread?